request-manager/graph/graph.go

```diff
@@ -44,14 +45,14 @@ type Node struct {
 	SequenceRetryWait string                 // The time to sleep between sequence retries
```

Comment:
Not sure the `Job job.Job` field is needed (above, not shown in GitHub diff). When we create a job node, we create the `job.Job`, then call its `Create` and `Serialize` methods, and then it's completely done; we don't have to keep a reference to it any longer. The important thing it does is mutate `jobArgs` and return a `[]byte` from its `Serialize` method.